From f53da6b510a3eea434ca57a9626b04328f2e2c2e Mon Sep 17 00:00:00 2001
From: Zheng Yejian <zhengyejian1@huawei.com>
Date: Tue, 23 Aug 2022 16:02:36 +0000
Subject: [PATCH 029/261] [Huawei] livepatch: Fix warning C_RULE_ID_BLANK_LINE

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187321

--------------------------------

CodeCheck reports that:
  Do not add blank lines on the start of a code block defined by braces.

Signed-off-by: Zheng Yejian <zhengyejian1@huawei.com>
---
 kernel/livepatch/core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/livepatch/core.c b/kernel/livepatch/core.c
index 1a3c0e4e300a..6e154307159b 100644
--- a/kernel/livepatch/core.c
+++ b/kernel/livepatch/core.c
@@ -1564,7 +1564,6 @@ static int klp_mem_prepare(struct klp_patch *patch)
 
 static void remove_breakpoint(struct klp_func *func, bool restore)
 {
-
 	struct klp_func_node *func_node = klp_find_func_node(func->old_func);
 	struct arch_klp_data *arch_data = &func_node->arch_data;
 
-- 
2.17.1

